spec/unit/services/users/omniauth_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+describe Users::Omniauth do
+  before do
+    @auth = OmniAuth.config.mock_auth[:github]
+  end
```

Comment:
@sinpai as a convention we do
```ruby
let(:auth) { OmniAuth.config.mock_auth[:github] }
```
instead of 
```ruby
before do
  @auth = OmniAuth.config.mock_auth[:github]
end
```